rts/Game/UI/GuiHandler.cpp

```diff
@@ -168,6 +169,8 @@ bool CGuiHandler::EnableLuaUI(bool enableCommand)
 		}
 	}
 
+	// TODO: Move to own console command?
+	RmlGui::Reload();
```

Comment:
What does `RmlGui::Reload()` effectively do? I've read the function's code, but I'm interested in user-facing effects.

Reply:
By reloading the RmlUi library, cached data is cleared.

Stylesheets and images most notably.

Of course something then needs to notify the code to reload the Rml based guis (typically a Widget from the LuaUI environment)...

Seems that the RmlGui::Reload() call needs to stay where it is then.

Reloading LuaUI or Rml separately from the other looks like it's just gonna break things